sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -612,9 +626,20 @@ inline pi_result piTearDown(void *PluginParameter) {
 inline pi_result piPlatformsGet(pi_uint32 NumEntries, pi_platform *Platforms,
                                 pi_uint32 *NumPlatforms) {
 
-  urInit(0);
+  urInit(0, nullptr);
+  // We're not going through the UR loader so we're guaranteed to have exactly

```

Comment:
@callumfare : We take this path even if we use the ur_loader with SYCL_PREFER_UR=1. are you saying here we dont?